tests/vm_test.go

```diff
@@ -656,19 +586,10 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 		})
 
 		It("should create vm revision when starting vm", func() {
-			newVM := newVirtualMachine(true)
-			vmCpy := newVM.DeepCopy()
-			vmCpy.Status = v1.VirtualMachineStatus{}
-
-			var vmi *v1.VirtualMachineInstance
-			Eventually(func() bool {
-				vmi, err = virtClient.VirtualMachineInstance(newVM.Namespace).Get(newVM.Name, &k8smetav1.GetOptions{})
-				if errors.IsNotFound(err) {
-					return false
-				}
-				Expect(err).ToNot(HaveOccurred())
-				return vmi.Status.Phase == v1.Running
-			}, 240*time.Second, 1*time.Second).Should(BeTrue())
+			vmi := libvmi.NewCirros()
+			vmi.Namespace = util.NamespaceTestDefault
+			newVM := createVM(virtClient, vmi, true)
+			vmi = waitForVMIRunning(virtClient, vmi)
```

Comment:
Since you touch this already:

You can use `Eventually(ThisVMI(vmi), 120).To(BeRunning())`. That integrates more nicely with the test framework than all those helper functions (like `EventuallyWithContext` can be used) 

We don't have one for `BeReady()` yet, but you could add one.

The benefit is also that it also allows combining conditions ad-hoc when needed, e.g.:


```
Eventually(ThisVMIWith(vm.Namespace, vm.Name), 160).Should(And(BeRunning(), BeOwned()))
```